tests/testsuite.src/used_binaries.at

```diff
@@ -1011,3 +1011,63 @@ AT_CHECK([$COBC -fdiagnostics-plain-output -fdiagnostics-show-caret -Wno-others
 
 AT_CLEANUP
 
+
+AT_SETUP([check include header file])
+AT_KEYWORDS([-include])
+
+AT_DATA([file.h], [
+extern void f(char *, long );
+])
+AT_DATA([prog.cob], [
+       IDENTIFICATION   DIVISION.
+       PROGRAM-ID.      prog.
+       PROCEDURE        DIVISION.
+           CALL "f" USING "Hello".
+])
+
+# No check, program seems correct
+
+AT_CHECK([$COBC -m -fstatic-call prog.cob])
+
+# We ignore the error output, as it depends on the C compiler in use
+
+AT_CHECK([$COBC -m --include "$PWD/file.h" -fstatic-call prog.cob], [1], [], [ignore])
+AT_DATA([prog2.cob], [
+       IDENTIFICATION   DIVISION.
+       PROGRAM-ID.      prog.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+       01 long USAGE BINARY-C-LONG.
+       PROCEDURE        DIVISION.
+           CALL "f" USING "Hello" BY VALUE long RETURNING NOTHING.
+])
+
+AT_CHECK([$COBC -m --include "$PWD/file.h" -fstatic-call prog2.cob])
+
+AT_CHECK([$COBC -I . -m --include "file.h" -fstatic-call prog2.cob])
```

Comment:
Mainly wondering: Is that better than using what we normally do: `, [0], [], [])`?
If yes we should someday apply this globally.

Hm, and for "being identical", _maybe_ add it back in the meanwhile?

Reply:
Yes, it is supposed to be equivalent.